src/main/resources/application-sqlgen.yml

```diff
@@ -0,0 +1,14 @@
+spring:
```

Comment:
this config is to allow us to easily see what hibernate generates for new entities instead of just writing it all ourselves from scratch.